common/libs/units.ts

```diff
@@ -103,7 +103,7 @@ const fromTokenBase = (value: TokenValue, decimal: number) =>
 const toTokenBase = (value: string, decimal: number) =>
   TokenValue(convertedToBaseUnit(value, decimal));
 
-const isEtherUnit = (unit: string) => unit === 'ether' || unit === 'ETH';
+const isEtherUnit = (unit: string) => unit === 'ETH';
```

Comment:
this should be used everywhere instead of manually checking for `unit === 'ETH'` so we only need to change one function if we ever need to deal with different base units (like EXP / UBIQ). I'm not too sure if we still need to check for `ether`, as API calls or node calls might return that instead of `ETH`

Reply:
good catch, yea `isEtherUnit()` should just be used everywhere and replace the one off conditions.